docs/api/checker.rst

```diff
@@ -906,7 +906,11 @@ Classic Classes
    >>> s = repr(C)
    >>> int(C.__module__ == __name__)
    1
-   >>> len(C.__bases__)
+
+Note that these are really only classic on Python 2:
+
+   >>> import sys
+   >>> len(C.__bases__) + (1 if sys.version_info[0] == 2 else 0)
    1
```

Comment:
(I also do not understand this one.)

Reply:
Above is `class C: pass`. On Python 2 that has no `__bases__`, but on Python 3, that's the same as `class C(object): pass`.

Reply:
Perhaps it would be clearer if you spelled it as

    >>> len(C.__bases__) == (0 if sys.version_info[0] == 2 else 1)
    True

Reply:
How about:
```pycon
>>> isinstance(C, getattr(types, 'ClassType', type))
True
```

Reply:
Oh, that doesn't work becasue `C` is proxied at this point.